recipes/occt/all/conanfile.py

```diff
@@ -40,25 +42,30 @@ def _patch_sources(self):
             '''project (OCCT)
                 include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
                 conan_basic_setup()''')
```

Comment:
```suggestion
                conan_basic_setup(TARGETS)''')
```

Reply:
The name here should be the name of the library but not the targets. I injected the Conan targets into another CMake file in the source. See https://github.com/conan-io/conan-center-index/pull/4094/files#diff-88b29f411668bc08514944d7b3f1487be09c15f8384132f4e3df5f69f064e6c9R49